modules/dynamic-plugins/ref-rh-supported-plugins.adoc

```diff
@@ -1,9 +1,9 @@
 // This page is generated! Do not edit the .adoc file, but instead run rhdh-supported-plugins.sh to regen this page from the latest plugin metadata.
 // cd /path/to/rhdh-documentation; ./modules/dynamic-plugins/rhdh-supported-plugins.sh; ./build/scripts/build.sh; google-chrome titles-generated/main/plugin-rhdh/index.html
 
-= {company-name} and partner plugins
+= {company-name} supported plugins
 
-{company-name} and its partners support the following 15 plugins:
+{company-name} support the following 17 plugins:
 
 [%header,cols=4*]
```

Comment:
Question/suggestion: since there are new plugins added, would it be helpful for users if there is a column that can add "New" so it is clear what plugins have been added?

Reply:
We're trying to reduce the number of columns and adding a 'new' label would need to be done manually after the table has been generated by the script referred to in other comments.

But I see your point, and the new plugins should be mentioned in the Release Notes. 

c.c. @themr0c

Reply:
or we could update the script to add stuff -- that's the reason the script exists. Just open a JIRA and I'll make the changes requested.

Reply:
I'm not sure we need this column if the new plugins are mentioned in the RNs.

Reply:
true, just means you need to do some manual diff (perhaps comparing the csv files) to see what's new

Reply:
To clarify, for my own edification: as a user, I will need to look at release notes and look at this table in order to know which plugins are new for a given release.

Reply:
Could we not LIST the new plugins in the RN doc, with a LINK to the tables in the Plugins Guide? 

So basically https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html-single/release_notes_for_red_hat_developer_hub_1.2/index#new-plugins-included-in-red-hat-developer-hub-1-2 with a link to https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html-single/configuring_plugins_in_red_hat_developer_hub/index#idm140684728973296 (as we have done in 1.2)

But with the updated content for 1.3.

Reply:
Yes @nickboldt I think we can maintain the approach we took for 1.2 i.e. listing new plugins in the RNs.